example/ktlint-baseline.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<baseline version="1.0">
```

Comment:
hm, does "baseline" mean "ignore the following ones because they were present when the linter was installed"?

Reply:
Yup, it is to ignore known violations. I've generated a baseline with some violations for `hello.kt` and included a few exceptions to illustrate it.

Reply:
Perfect thanks